diff --git a/adm_program/modules/categories/categories.php b/adm_program/modules/categories/categories.php
index cb519e5..d32d63d 100644
--- a/adm_program/modules/categories/categories.php
+++ b/adm_program/modules/categories/categories.php
@@ -22,7 +22,7 @@ require_once('../../system/common.php');
 require_once('../../system/login_valid.php');
 
 // Initialize and check the parameters
-$getType  = admFuncVariableIsValid($_GET, 'type', 'string',  array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF')));
+$getType  = admFuncVariableIsValid($_GET, 'type', 'string',  array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF','AWA')));
 $getTitle = admFuncVariableIsValid($_GET, 'title', 'string', array('defaultValue' => $gL10n->get('SYS_CATEGORY')));
 
 // Modus und Rechte pruefen
@@ -42,6 +42,10 @@ elseif($getType == 'DAT' && $gCurrentUser->editDates() == false)
 {
     $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
 }
+elseif($getType == 'AWA' && $gCurrentUser->editUsers() == false)
+{
+    $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
+}
 
 // set module headline
 $headline = $gL10n->get('SYS_ADMINISTRATION_VAR', $getTitle);
diff --git a/adm_program/modules/categories/categories_function.php b/adm_program/modules/categories/categories_function.php
index c240668..3b0ed42 100644
--- a/adm_program/modules/categories/categories_function.php
+++ b/adm_program/modules/categories/categories_function.php
@@ -27,7 +27,7 @@ require_once('../../system/login_valid.php');
 
 // Initialize and check the parameters
 $getCatId    = admFuncVariableIsValid($_GET, 'cat_id', 'numeric');
-$getType     = admFuncVariableIsValid($_GET, 'type', 'string',  array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF')));
+$getType     = admFuncVariableIsValid($_GET, 'type', 'string',  array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF','AWA')));
 $getMode     = admFuncVariableIsValid($_GET, 'mode', 'numeric', array('requireValue' => true));
 $getTitle    = admFuncVariableIsValid($_GET, 'title', 'string', array('defaultValue' => $gL10n->get('SYS_CATEGORY')));
 $getSequence = admFuncVariableIsValid($_GET, 'sequence', 'string', array('validValues' => array('UP', 'DOWN')));
@@ -49,6 +49,10 @@ elseif($getType == 'DAT' && $gCurrentUser->editDates() == false)
 {
     $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
 }
+elseif($getType == 'AWA' && $gCurrentUser->editUsers() == false)
+{
+    $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
+}
 
 // create category object
 $category = new TableCategory($gDb);
@@ -209,4 +213,4 @@ elseif($getMode == 4)
     $category->moveSequence($getSequence);
     exit();
 }
-?>
\ No newline at end of file
+?>
diff --git a/adm_program/modules/categories/categories_new.php b/adm_program/modules/categories/categories_new.php
index b05093d..09885c3 100644
--- a/adm_program/modules/categories/categories_new.php
+++ b/adm_program/modules/categories/categories_new.php
@@ -24,7 +24,7 @@ require_once('../../system/login_valid.php');
 
 // Initialize and check the parameters
 $getCatId = admFuncVariableIsValid($_GET, 'cat_id', 'numeric');
-$getType  = admFuncVariableIsValid($_GET, 'type', 'string', array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF')));
+$getType  = admFuncVariableIsValid($_GET, 'type', 'string', array('requireValue' => true, 'validValues' => array('ROL', 'LNK', 'USF', 'DAT', 'INF','AWA')));
 $getTitle = admFuncVariableIsValid($_GET, 'title', 'string', array('defaultValue' => $gL10n->get('SYS_CATEGORY')));
 
 // Modus und Rechte pruefen
@@ -44,6 +44,10 @@ elseif($getType == 'DAT' && $gCurrentUser->editDates() == false)
 {
     $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
 }
+elseif($getType == 'AWA' && $gCurrentUser->editUsers() == false)
+{
+    $gMessage->show($gL10n->get('SYS_NO_RIGHTS'));
+}
 
 // set headline of the script
 if($getCatId > 0)
@@ -134,4 +138,4 @@ $form->addHtml(admFuncShowCreateChangeInfoById($category->getValue('cat_usr_id_c
 $page->addHtml($form->show(false));
 $page->show();
 
-?>
\ No newline at end of file
+?>
diff --git a/adm_program/modules/profile/profile.php b/adm_program/modules/profile/profile.php
index b46ec3b..c50f2d5 100644
--- a/adm_program/modules/profile/profile.php
+++ b/adm_program/modules/profile/profile.php
@@ -818,7 +818,10 @@ if($gPreferences['profile_show_extern_roles'] == 1
         }
     }
 }
-
+if(is_file('../../../adm_plugins/awards/awards_profile_addin.php'))
+{
+	require_once('../../../adm_plugins/awards/awards_profile_addin.php');
+}
 // show informations about user who creates the recordset and changed it
 $page->addHtml(admFuncShowCreateChangeInfoById($user->getValue('usr_usr_id_create'), $user->getValue('usr_timestamp_create'), $user->getValue('usr_usr_id_change'), $user->getValue('usr_timestamp_change')));
 
 
